airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/ErrorMessageFactory.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors;
+
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.DEFAULT;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.GCS;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.MONGO;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.MSSQL;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.MYSQL;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.ORACLE;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.POSTGRES;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.REDSHIFT;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.SNOWFLAKE;
+
+import io.airbyte.integrations.base.errors.messages.DefaultErrorMessage;
+import io.airbyte.integrations.base.errors.messages.ErrorMessage;
+import io.airbyte.integrations.base.errors.messages.GcsErrorMessage;
+import io.airbyte.integrations.base.errors.messages.MongoDbErrorMessage;
+import io.airbyte.integrations.base.errors.messages.MssqlErrorMessage;
+import io.airbyte.integrations.base.errors.messages.MysqlErrorMessage;
+import io.airbyte.integrations.base.errors.messages.OracleErrorMessage;
+import io.airbyte.integrations.base.errors.messages.PostgresErrorMessage;
+import io.airbyte.integrations.base.errors.messages.RedshiftErrorMessage;
+import io.airbyte.integrations.base.errors.messages.SnowflakeErrorMessage;
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+import java.util.Map;
+
+public class ErrorMessageFactory {
+
+  private final static Map<ConnectorName, ErrorMessage> MAP = Map.of(MSSQL, new MssqlErrorMessage(),
```

Comment:
Minor nit: `MAP` as a constant doesn't seem like the best naming convention for understanding what the constant stands for. Alternative name might be something like `CONNECTOR_ERROR_MESSAGE_MAP` 

Also for the order of the `<Database_Name>ErrorMessage()` below, wondering if keeping it lexicographically sorted like the imports may help to quickly parse whether a Database already extends the `ErrorMessage` class you've created

Reply:
renamed to CONNECTOR_ERROR_MESSAGE_MAP